src/components/TopBar/TopBarContent.tsx

```diff
@@ -90,7 +99,12 @@ export default function TopBarContent(props: TopBarProps): JSX.Element | null {
         )}
       </Grid>
 
-      <Grid item xs={6} className={`${classes.center} logo`} />
+      <Grid
+        item
+        xs={6}
+        className={`${classes.center} ${classes.clickableLogo} logo`}
```

Comment:
Can we add cursor: pointer here? In desktop the <Link> is already adding it

Reply:
for mobile will it matter? i will add it. just curious

Reply:
Oh, you are right, not needed. I was testing with the inspector